analytic_plan/views/account_analytic_account_view.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_analytic_account_tree_inh" model="ir.ui.view">
+        <field name="name">account.analytic.account.tree</field>
+        <field name="model">account.analytic.account</field>
+        <field name="type">tree</field>
+        <field name="field_parent">child_complete_ids</field>
+        <field name="inherit_id"
+               ref="analytic.view_account_analytic_account_list"/>
+        <field name="arch" type="xml">
+            <field name="balance" position="after">
+                <field name="debit_plan"/>
+                <field name="credit_plan"/>
+                <field name="balance_plan"/>
+                <button string="Planned cost lines"
+                        name="open_plan_cost_tree_view"
+                        type="object"
+                        icon="gtk-open"/>
```

Comment:
Icon is not working and action is missing.

I don't know if it is OK for me to leave feedback here?

Reply:
It is fine @CasVissers Thank you for reviewing. However this PR is kind of a mess. I think it is better to merge this and then fix every module in a single Pull Request